cogni-sem/src/test/java/zone/cogni/sem/jena/SparqlUtilsTest.java

```diff
@@ -0,0 +1,58 @@
+package zone.cogni.sem.jena;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import zone.cogni.libs.sparqlservice.impl.JenaModelSparqlService;
+
+public class SparqlUtilsTest {
+  private static final String URI = "https://fedlex.data.admin.ch/eli/oc/2023/423/legal-analysis";
+  private static final String GRAPH_URI = URI + "/graph";
+  private final JenaModelSparqlService jenaModelSparqlService = new JenaModelSparqlService();
+
+  @Test
+  public void test() {
+    testEscape("A\ncomment"); // test newline
```

Comment:
add "mixed" testcases 
things like this: "Bla\\n" and "Bla\\\n"

Reply:
added a couple of them